README.md

```diff
@@ -357,6 +357,46 @@ TrySafe::fromCallable(fn() => $getTheUltimateAnswerOrThrow(true))
  
 _Disclaimer:_ As you probably noticed, we reflect possible exception in return type now, but on the other side, we've lost the information that wrapped success value is `float`.  This applies also to `Option`, `ArrayList` etc. Unfortunately there is no silver bullet solution, until PHP have the generics implemented (but hey, have look at [phpstan generics templates](https://medium.com/@ondrejmirtes/generics-in-php-using-phpdocs-14e7301953)). 
 
+#### TrySafe recovery
+
+We have already learned, that `TrySafe` can server to chaining dependant operations that can fail (via `flatMap`). How about if we want to have some fallback / recovery in the middle of that chain?
+
```

Comment:
Can serve for chaining? (But not sure about this one)

Reply:
I have changed it to:

```
We have already learned, that `TrySafe` can be used for chaining dependant operations that can fail (via `flatMap`).
```

Reply:
dependant -> dependent

Reply:
:+1: